include/silo/zstdfasta/zstdfasta_table.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include <string>
+
+namespace duckdb {
+struct Connection;
+}
+
+namespace silo {
```

Comment:
True, then I should also rename the classes to not contain the namespace at the start again. Do you think this would mess up the change list too much and I do it separately? (And I do #139 in the near future)